src/Template/Element/legal/pl_PL/generalTermsAndConditions.ctp

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * FoodCoopShop - The open source software for your foodcoop
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @since         FoodCoopShop 2.1.0
```

Comment:
in this case your changings would be part of the version 2.5.0 which will be released in june 2019. whenever a new file is created, this is the @since version. at least I handled it like this with all other files.